Sources/Confidence/PayloadMerger.swift

```diff
@@ -6,9 +6,14 @@ internal protocol PayloadMerger {
 
 internal struct PayloadMergerImpl: PayloadMerger {
     func merge(context: ConfidenceStruct, message: ConfidenceStruct) -> ConfidenceStruct {
-        var map: ConfidenceStruct = context
-        map += message
-        return map
+        let messageContextStruct = message["context"]?.asStructure() ?? [:]
```

Comment:
In JS we throw an exception if the user put a `context` key in the `message`/`data`. IMO that is preferred since the developer used the API incorrectly (it is not allowed to have a `context` key in the `message/data`). It is better to let the developer know this instead of secretly merge and remove some of their data.

Reply:
https://github.com/spotify/confidence-sdk-js/commit/2f73b3b519082fa58a64de3d3be957571dc72a00#diff-c4db671881d7ec70956d14c602874d8da8fb333e50987901a6f60c66c95ff506R73